src/specBuilder/trendline/trendlineUtils.ts

```diff
@@ -422,25 +456,38 @@ export const getTrendlineScales = (props: LineSpecProps | BarSpecProps): Scale[]
  * @param method
  * @returns boolean
  */
-const isPolynomialMethod = (method: TrendlineMethod): boolean =>
-	method.startsWith('polynomial-') || ['linear', 'quadratic'].includes(method);
+const isAggregateMethod = (method: TrendlineMethod): method is AggregateMethod =>
+	['average', 'median'].includes(method);
+
+// export const isLineTypeArray = (lineTypes: LineTypes): lineTypes is LineType[] => {
+// 	return lineTypes.some((lineType) => typeof lineType === 'string' || isStrokeDashArray(lineType));
+// };
```

Comment:
Did this guy just get forgotten? :)

Reply:
Good catch. I pasted that there as a reference of how to right a typeguard.